org.gnome.design.Emblem.json

```
@@ -0,0 +1,90 @@
+{
+    "app-id": "org.gnome.design.Emblem",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41rc",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "emblem",
+    "finish-args" : [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--talk-name=org.a11y.Bus"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin"
+    },
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name" : "libadwaita",
+            "buildsystem" : "meson",
+            "cleanup": [
+                "/include",
+                "/lib/pkgconfig"
+            ],
+            "config-opts": [
+                "-Dgtk_doc=false",
+                "-Dtests=false",
+                "-Dexamples=false",
+                "-Dvapi=false"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/GNOME/libadwaita.git",
+                    "commit": "03f159488ec3b8e3ea4c3c2daa9c408b071d512d"
+                }
+            ],
+            "modules" : [
+                {
+                    "name" : "libsass",
+                    "buildsystem" : "meson",
+                    "cleanup": ["*"],
+                    "sources" : [
+                        {
+                            "type" : "git",
+                            "url" : "https://github.com/lazka/libsass.git",
+                            "branch" : "meson",
+                            "commit": "302397c0c8ae2d7ab02f45ea461c2c3d768f248e"
```

```suggestion
                            "commit": "302397c0c8ae2d7ab02f45ea461c2c3d768f248e"
```
branch + commit will fail to build as soon as a new commit is pushed to the repository. Please only a commit if there is no tag.